src/components/tables/Captures/index.tsx

```diff
@@ -0,0 +1,117 @@
+import {
+    Box,
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableHead,
+    TableRow,
+    Typography,
+} from '@mui/material';
+import Error from 'components/shared/Error';
+import Empty from 'components/tables/Captures/Empty';
+import Rows from 'components/tables/Captures/Rows';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { TABLES } from 'services/supabase';
+import { useQuery, useSelect } from 'supabase-swr';
+
+interface Discovers {
+    'capture_name': string;
+    'updated_at': string;
+    'job_status->>type': string;
+    'id': string;
+}
```

Comment:
Need to eventually figure a pattern out for these. We need these types to help with getting safer filters and other query builders. However, when you do something like parse the JSON in the query the data that comes back does not match the query. 

For example `job_status->>type` comes back in the data just as `type`... which makes sense. This is why I am still leaving the `data` prop on the response as an any for now.